package.json

```diff
@@ -148,6 +148,9 @@
     "request": "2.84.0",
     "rimraf": "2.6.2",
     "rocambole": "0.7.0",
+    "rollup": "^0.57.1",
```

Comment:
If it's in our specific package.json, why doe it also need to be here?

Reply:
yep, removed it from here